### DISCUSSION TITLE
Looking for someone to fine tune the TTS model

### DISCUSSION THREAD

**Discussion by gunpal5** [coqui-ai/TTS#2573] [General]
Hello,

Thank you for putting forward such a great project in public domain. We want to use TTS as an API server for our existing app which create voiceovers for videos. If it is allowed by the license.

If it is allowed by the license. Then We are looking for someone can fine tune the TTS model for our application. We are looking for someone who can create 4-5 voices which are comparable to other commercially available products, if possible.

Thank you

**Comment by marisbasha** [comment]
Contact me m@niv.al
